python_src/src/lamp_py/performance_manager/l0_gtfs_static_mod.py

```diff
@@ -6,7 +6,6 @@
     StaticStopTimes,
     StaticStops,
     StaticTrips,
-    TempStaticHeadwaysGen,
 )
```

Comment:
Drop this TempStaticHeadwaysGen table, improved table indexes make this unnecessary.

Reply:
It seems to be at least equally performative in my testing.